iOS/cmux-remote/docs/architecture.md

```
@@ -0,0 +1,131 @@
+# cmux-remote architecture
+
+## Layered overview
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)